src/main/java/org/example/c02controlflow/generate/PascalTriangle.java

```diff
@@ -17,20 +15,20 @@
  */
 
 public class PascalTriangle {
-    public static void main(String[] args) {
-        Scanner sc = new Scanner(System.in);
-        System.out.print("Enter number of rows: ");
-        int numberOfRows = sc.nextInt();
-        for (int i = 1; i <= numberOfRows; i++) {
-            for (int j = 0; j <= numberOfRows - i; j++) {
-                System.out.print(" ");
+    public static String builtTriangle(int number) {
+        String triangle = "";
+        for (int i = 1; i <= number; i++) {
+            for (int j = 0; j <= number - i; j++) {
+                triangle += " ";
             }
             int l = 1;
             for (int k = 1; k <= i; k++) {
-                System.out.print(l + " ");
+                triangle += l + " ";
```

Comment:
alternative use:
```java
triangle += l;
 if(k<i)
     triangle  += " ";
```
to avoid extra space at the end of the line
then you can remove 
```triangle = triangle.substring(0, triangle.length() - 1);```

Reply:
Done